spec/support/schemas/claims_api/v2/veterans/claims/claim.json

```diff
@@ -2,28 +2,26 @@
   "$schema": "http://json-schema.org/draft-04/schema#",
   "type": "object",
   "additionalProperties": false,
-  "required": ["id", "type", "status", "@links"],
+  "required": [
+    "id",
+    "@links",
+    "contentionList",
+    "dateFiled",
+    "decisionLetterSent",
+    "developmentLetterSent",
+    "documentsNeeded",
+    "endProductCode",
+    "status",
+    "supportingDocuments",
+    "type",
+    "vaRepresentative"
+  ],
```

Comment:
the semantics of "required" are a little odd here, because it's for a GET response. I'm interpreting that as "we guarantee these values will be present in the response," which in this case meant I added everything outside of `requestedDecision`, which could have a null value

Reply:
If we're ok with that interpretation, then we should update the [claims list schema required values](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/support/schemas/claims_api/v2/veterans/claims/claims.json#L5) in a separate PR

Reply:
It's late in the day and my brain is probably just tired, but I want to make sure that we're not ever optionally including attributes in the response, right?  Like, even if an attribute could have a null/blank value, we're still going to include that attribute even when it's null/blank aren't we?

Reply:
> even if an attribute could have a null/blank value, we're still going to include that attribute even when it's null/blank aren't we?

Yeah. [According to the JSON schema docs](https://json-schema.org/understanding-json-schema/reference/object.html?highlight=required#required-properties), it defaults to properties not being required (which I don’t think we need to care too much about anyway in the context of a GET request. this is just how we build out the responses w/ rswag, so it's more just a means to an end):

> By default, the properties defined by the properties keyword are not required. However, one can provide a list of required properties using the required keyword.

Reply:
Removed required [here](https://github.com/department-of-veterans-affairs/vets-api/pull/9102/commits/24d5a62175c3ebed7a41a3ee321a7e15569971b8)

Reply:
Probably want RSWAG to regenerate the V2 docs after removing the "required" stuff.